diff -urN graphviz-2.16.1.orig/plugin/pango/gvtextlayout_pango.c graphviz-2.16.1/plugin/pango/gvtextlayout_pango.c
--- graphviz-2.16.1.orig/plugin/pango/gvtextlayout_pango.c	2007-11-09 17:31:43.000000000 -0600
+++ graphviz-2.16.1/plugin/pango/gvtextlayout_pango.c	2008-03-03 09:55:01.000000000 -0600
@@ -154,8 +154,8 @@
 	logical_rect.height = 0;
 
     textlayout_scale = POINTS_PER_INCH / (FONT_DPI * PANGO_SCALE);
-    para->width = logical_rect.width * textlayout_scale;
-    para->height = logical_rect.height * textlayout_scale;
+    para->width = ROUND(logical_rect.width * textlayout_scale);
+    para->height = ROUND(logical_rect.height * textlayout_scale);
 
     /* The y offset from baseline to 0,0 of the bitmap representation */
     iter = pango_layout_get_iter (layout);
