diff -up graphviz-2.30.1/config/config_perl.pl.orig graphviz-2.30.1/config/config_perl.pl
--- graphviz-2.30.1/config/config_perl.pl.orig	2013-02-14 14:27:39.000000000 +0100
+++ graphviz-2.30.1/config/config_perl.pl	2013-08-05 14:38:06.409121360 +0200
@@ -14,7 +14,7 @@ if ($ARGV[0] eq "PERL_INSTALL_DIR") {
 
 	my $d;
 
-	foreach $d qw(installvendorarch vendorarch installsitearch sitearch) {
+	foreach $d (qw(installvendorarch vendorarch installsitearch sitearch)) {
 		if (exists($Config{$d}) and defined($Config{$d}) and
 			($Config{$d} ne '') ) {
 			print "$Config{$d}";
