Index: lib/common/Makefile.am
===================================================================
RCS file: /home/cvsroot/graphviz2/lib/common/Makefile.am,v
retrieving revision 1.46
retrieving revision 1.47
diff -u -r1.46 -r1.47
--- lib/common/Makefile.am	26 Jul 2007 13:53:25 -0000	1.46
+++ lib/common/Makefile.am	4 Sep 2007 23:39:07 -0000	1.47
@@ -1,4 +1,4 @@
-# $Id: Makefile.am,v 1.46 2007/07/26 13:53:25 erg Exp $ $Revision: 1.46 $
+# $Id: Makefile.am,v 1.47 2007/09/04 23:39:07 ellson Exp $ $Revision: 1.47 $
 ## Process this file with automake to produce Makefile.in
 
 AM_CPPFLAGS = \
@@ -14,11 +14,10 @@
 AM_CFLAGS = -D_BLD_common=1
 endif
 
-pkginclude_HEADERS = geom.h color.h types.h textpara.h usershape.h
+pkginclude_HEADERS = arith.h geom.h color.h types.h textpara.h usershape.h
 noinst_HEADERS = render.h utils.h memory.h \
-	geomprocs.h colorprocs.h colortbl.h entities.h \
-	logic.h arith.h globals.h \
-	const.h macros.h htmllex.h htmltable.h pointset.h
+	geomprocs.h colorprocs.h colortbl.h entities.h globals.h \
+	logic.h const.h macros.h htmllex.h htmltable.h pointset.h
 noinst_LTLIBRARIES = libcommon_C.la
 
 if WITH_CODEGENS
